python/dlr/api.py

```diff
@@ -17,6 +17,10 @@ def get_input_names(self):
     def get_input(self, name, shape=None):
         raise NotImplementedError
 
+    @abc.abstractmethod
+    def get_output_names(self):
```

Comment:
Per discussion with @apivovarov we're exposing this API and leave the TVM/Treelite impl as placeholder. We should open a discussion thread and possibly maintain a table showing which public DLR API corresponds to which API for each underlying runtime.